chore(github-integration-plugin): Updating docs with GitHub Integration plugin trigger types (#2215)
armory/docs: content/en/plugins/github-integration/use.md

{{< figure src="/images/plugins/github/workflowFinish.png" >}}

* **Type**: (Required) Select **GitHub Workflow Trigger**
* **Github Account**: (Required) Select the GitHub Account; this is one of the accounts you configured when you installed the plugin
* **Organization**: (Required) Select the organization associated with the **Github Account**
* **Repository**: (Required) Select the repository that contains the workflow
* **Workflow**: (Required) Select the name of the workflow that Spinnaker should monitor

> * **Branch**: (Optional) If specified, only pushes to the branches that match this Java Regular Expression will be triggered. Leave empty to trigger builds for every branch.

Reviewer: ```suggestion
* **Branch**: (Optional) If specified, only pushes to the branches that match this Java Regular Expression are triggered. Leave empty to trigger builds for every branch.
```
